src/components/SearchTextareaField/SearchTextareaField.test.js

```diff
@@ -51,4 +55,12 @@ describe('Given SearchTextareaField', () => {
       expect(onChange).toHaveBeenCalled();
     });
   });
+
+  describe('when user press "Enter" button', () => {
+    it('should handle form submit', async () => {
+      onKeyDown();
+
+      expect(onSubmitSearch).toHaveBeenCalled();
+    });
+  });
```

Comment:
Please, write test by the template:
1. Render the component
2. Make all that you can to get expected result
3. Check if you test pass or not

```suggestion
  describe('when user press "Enter" button', () => {
    it('should handle form submit', () => {
      // create a mock of submit function
      const mockOnSerachSubmit = jest.fn();
      // 1. render component
      const { getByTestId } = renderSearchTextareaField({
        onSearchSubmit: mockOnSearchSubmit, // set mock function to the component
      }); 
  
      // 2. implementation part
      fireEvent.change(getByTestId('search-textarea'), { target: { value: 'test' } }); // set value to the textarea  
      fireEvent.keyDown(getByTestId('search-textarea'), { 
        keyCode: 13,
      }); // press enter in the textarea

      // 3.here you check test
      expect(mockOnSubmitSearch).toHaveBeenCalled(); // check if onSearchSubmit handled
    });
  });
```